library/core/src/cell.rs

```diff
@@ -209,6 +209,10 @@ pub use once::OnceCell;
 
 /// A mutable memory location.
 ///
+/// # Memory layout
+///
+/// `Cell<T>` shares the same memory layout and caveats as `UnsafeCell<T>`.
```

Comment:
This would be clearer if it repeated the first sentence from the `UnsafeCell` docs:

> `UnsafeCell<T>` has the same in-memory representation as its inner type `T`.

For the rest it can refer to the `UnsafeCell` docs (with a doc-link).

Reply:
Good point, I've done this now